site/content/resources/upgrading.md

```diff
@@ -9,6 +9,72 @@ This document describes the changes needed to upgrade your Contour installation.
 
 <div id="toc" class="navigation"></div>
 
+## Upgrading Contour 1.15.1 to 1.16.0
+
+Contour 1.16.0 is the current stable release.
+
+### Required Envoy version
+
+All users should ensure the Envoy image version is `docker.io/envoyproxy/envoy:v1.18.3`.
+
+Please see the [Envoy Release Notes][32] for information about issues fixed in Envoy 1.18.3.
+
+### The easiest way to upgrade is with Contour Operator (alpha)
+- Verify the operator is running v1.15.1, and it's deployment status is "Available=True".
+- Verify the status of all Contour custom resources are "Available=True".
+- Update the operator's image to v1.16.0:
+   ```bash
+   $ kubectl patch deploy/contour-operator -n contour-operator -p '{"spec":{"template":{"spec":{"containers":[{"name":"contour-operator","image":"docker.io/projectcontour/contour-operator:v1.16.0"}]}}}}'
+   ```
+- The above command will upgrade the operator. After the operator runs the new version, it will upgrade Contour.
+- Verify the operator and Contour are running the new version.
+- Verify all Contour custom resources are "Available=True".
```

Comment:
Maybe add a sentence explicitly stating that this is only relevant/valid for current users of the contour operator.